pkg/asset/agent/image/ignition.go

```diff
@@ -360,6 +373,53 @@ func addExtraManifests(config *igntypes.Config, extraManifests *manifests.ExtraM
 	}
 }
 
+func getOSImagesInfo() (*models.OsImage, error) {
+	ctx, cancel := context.WithTimeout(context.TODO(), 30*time.Second)
+	defer cancel()
+
+	st, err := rhcos.FetchCoreOSBuild(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	// Architecture currently hard-coded to x86_64
+	cpurArch := archName
```

Comment:
Let's make this a parameter, since we know it will have to be one eventually. Multi-arch folks are already working on ARM support in #6401.